pandas/_libs/tslibs/parsing.pyx

```diff
@@ -33,6 +34,24 @@ from dateutil.parser import parse as du_parse
 from pandas._libs.tslibs.ccalendar import MONTH_NUMBERS
 from pandas._libs.tslibs.nattype import nat_strings, NaT
 
+cdef extern from "errno.h":
+    int errno
+
+cdef extern from "../src/parser/tokenizer.h":
+    double xstrtod(const char *p, char **q, char decimal, char sci, char tsep,
+                   int skip_trailing)
+
+cdef extern from *:
+    char* PyUnicode_AsUTF8AndSize(object unicode, Py_ssize_t* length)
```

Comment:
Can’t cimport?

Reply:
No, this particular function is for some reason not exported in `cpython.pxd`, I don't know why...